### ISSUE TITLE
Unable to debug SourceGenerator, no output and no log files created

### ISSUE THREAD

**Issue by xumix** [NeVeSpl/NTypewriter#94]
Package versions:
```
    <PackageReference Include="NTypewriter.CodeModel">
      <Version>0.4.4.3</Version>
    </PackageReference>
    <PackageReference Include="NTypewriter.CodeModel.Functions">
      <Version>0.4.4.3</Version>
    </PackageReference>
    <PackageReference Include="NTypewriter.Editor.Config">
      <Version>0.4.4.3</Version>
    </PackageReference>
    <PackageReference Include="NTypewriter.SourceGenerator">
      <Version>0.2.5</Version>
    </PackageReference>
```
The error:
`CSC: Warning CS8785 : Generator 'NTypewriterSourceGenerator' failed to generate source. It will not contribute to the output and compilation errors may occur as a result. Exception was of type 'ArgumentNullException' with message 'Value cannot be null.`

How do I debug the SourceGenerator version or how do I enable full exception logging?

**Comment by NeVeSpl** [comment]
Does it not work on any project or only on a specific one?

**Comment by NeVeSpl** [comment]
what can I say, developing source generators is really painful.

**Comment by xumix** [comment]
> what can I say, developing source generators is really painful.

Any way to get logs? I can't find any that are mentioned in the docs

**Comment by NeVeSpl** [comment]
Logs are generated only if everything went well on SG side:
https://github.com/NeVeSpl/NTypewriter/blob/deda6ab19f3e73b3b6b070103a8cc611d0287e65/NTypewriter.SourceGenerator/SourceGenerator.cs#L99C20-L99C20

if there is a problem with SG side, you should look for exceptions and what is in VS output window:
https://github.com/NeVeSpl/NTypewriter/blob/deda6ab19f3e73b3b6b070103a8cc611d0287e65/NTypewriter.SourceGenerator/SourceGenerator.cs#L103

**Comment by NeVeSpl** [comment]
It was probably the same bug as #103, related to missing build output path, fixed in version v0.3.4.
